_posts/2017-06-06-Spring-IO-2017-The-Spring-ecosystem.md

```diff
@@ -0,0 +1,318 @@
+---
+layout: post
+authors: [matthias_caryn]
+title: 'Spring IO 2017: The Spring Ecosystem'
+image: /img/spring-ecosystem/springEcosystemHeaderImageJPG.jpg
+tags: [Spring, Ecosystem, Cloud, Conference]
+category: Spring
+comments: true
+---
+
+
+> When I was at Spring IO back in May, I was intrigued by a presentation given by Jeroen Sterkens. 
+There he talked about the Spring Ecosystem in 50 minutes. 
+As only having 50 minutes, he could not focus on all the projects and I wanted to get a feel what Pivotal Software has to offer in all its glory by getting to know all the main projects.
```

Comment:
I'd say "The Spring Team" instead of Pivotal Software, it emphasizes the opensource nature